src/win/util.c

```diff
@@ -504,8 +505,8 @@ uint64_t uv__hrtime(unsigned int scale) {
    * performance counter interval, integer math could cause this computation
    * to overflow. Therefore we resort to floating point math.
    */
-  double scaled_freq = (double)hrtime_frequency_ / scale;
-  double result = (double) counter.QuadPart / scaled_freq;
+  scaled_freq = (double)hrtime_frequency_ / scale;
```

Comment:
While you are at it, could you please add space between `(double)` and `hrtime_freqency_`?

Reply:
ok